### DISCUSSION TITLE
Allow for the use of hidden and Traceable together

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#15963] [Ideas]
After playing traceable for a decent amounf of time  (20 hours or so) in conjuction with others mods, i have come to the conclusion that there is no noteable reason to not make it compatible with hidden, 

whilst it is true that hidden has tradiotionally been an effect applied to hitcircles not approach circles, allowing traceable to override this when both are enabled would allow another mod combination for users to enjoy.

i would not be proposing this without consideration, but seeing as ive come close to setting some of my best plays on it i can conclude that Traceable is more than a gimmick, and the time that it would take to make them work together wouldnt be a waste

**Comment by bdach** [comment]
a mod shouldn't affect another mod's behaviour. that is inviting a new world of trouble and introduces an exponential complexity factor in handling all possible combinations.

if the two are to be compatible, the result of selecting both needs to be the exact sum of behaviour of both. which is to say, no things such as "if i put on hidden and then traceable, hidden will only hide approach circles".

**Reply by mk56-spn** [reply]
alright, is there anyway the desired behaviour can be achieved? as a subsetting? because its legitmately something id consider fun

**Reply by bdach** [reply]
hidden already has a setting to only hide approach circles. having HD with that setting + TC is fine. but again, there should be no interaction between the two - so enabling TC should not automatically enable that setting.

**Reply by mk56-spn** [reply]
except that does the exact opposite, what im trying to achieve here is fading  approaches, not no approach circles and a hitcircle like  the special hidden does

**Reply by bdach** [reply]
it would invariably have to be some sort of HD specific setting which can be used always, regardless of if TC is also enabled.

**Reply by mk56-spn** [reply]
ok understood, regardless of this thing i think there should be some sort of documentation of precedent somewhere so this question doesnt pop  up in exponentially increasing amounts of discussions once more fun mods roll in.
